doc/man3/OSSL_SERIALIZER_EVP_PKEY_to_bio.pod

```diff
@@ -0,0 +1,63 @@
+=pod
+
+=head1 NAME
+
+OSSL_SERIALIZER_EVP_PKEY_to_bio, OSSL_SERIALIZER_EVP_PKEY_to_fp
+- serialization of EVP_PKEY objects
+
+=head1 SYNOPSIS
+
+ #include <openssl/serializer.h>
+
+ int OSSL_SERIALIZER_EVP_PKEY_to_bio(BIO *out, const EVP_PKEY *pkey,
+                                     const char *propquery,
+                                     int want_domparams);
+ int OSSL_SERIALIZER_EVP_PKEY_to_fp(FILE *fp, const EVP_PKEY *pkey,
+                                    const char *propquery,
+                                    int want_domparams);
+
+=head1 SYNOPSIS
+
+OSSL_SERIALIZER_EVP_PKEY_to_bio() finds the L<OSSL_SERIALIZER(3)> that
+matches the assymetric key I<pkey> and the desired properties
```

Comment:
asymmetric

Reply:
I've rewritten much of the docs, and this seems to be gone now